code_challenges.rb

```diff
@@ -20,14 +32,58 @@
 album3 = 'Abbey Road'
 # Expected output: 'bby Rd'
 
+#create method called theBeatles
+#method will take in string
+#change characters in sting to lowercase with .downcase
+#delete vowels with .delete "aeiou"
+#the beatles rock!
+def theBeatles string
+    string.downcase.delete "aeiou"
+end
```

Comment:
Close, the expected output calls for some casing to be preserved-- but I like where your head is at trying to clean the data so that your delete works consistently